root66_website/front_end/templates/front_end/clubs.html

```diff
@@ -1,35 +1,93 @@
+            height:200px;
+            border-radius:5%;
+        }
+        .center {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            width: 50%;
+        }
+    </style>
+</head>
 
-<div class="button">
+    <!-- Our Clubs and Teams section -->
+<div class="container mt-5 text-center">
+    <!-- Image grid -->
+    <div class="row mt-4">
+        <!-- WICYS -->
+        <div class="col-md-3">
+            <div class="image-container">
+                <a href="{% url 'front_end:wicys' %}">
+                    <img class="logo" src="{% static 'img/wicys.jpg' %}" alt="WiCyS" class="img-fluid">
+                    <div class="overlay">
+                        <p>WiCyS Club</p>
+                    </div>
+                </a>
+            </div>
+        </div>
 
-    <div id="button" style="text-align: center;">
-        <button onclick="show()">
-        <img width="200" height="200" src="{% static 'img/red_button.jpg' %}">
-        </button>
-    </div>
+        <!-- CTF -->
+        <div class="col-md-3">
+            <div class="image-container">
+                <a href="{% url 'front_end:ctf' %}">
+                    <img class="logo" src="{% static 'img/TUCTF.jpg' %}" alt="CTF" class="img-fluid">
+                    <div class="overlay">
+                        <p>CTF Club</p>
+                    </div>
+                </a>
+            </div>
+        </div>
 
-    <div id="clubs" class="hidden" style="display: none; text-align: center;">
-        <a href="{% url 'front_end:wicys' %}">
-            <p>WICYS</p>
-        </a>
+        <!-- CCDC -->
+        <div class="col-md-3">
+            <div class="image-container">
+                <a href="{% url 'front_end:ccdc' %}">
+                    <img class="logo" src="{% static 'img/ccdc.jpg' %}" alt="CCDC" class="img-fluid">
+                    <div class="overlay">
+                        <p>CCDC Team</p>
+                    </div>
+                </a>
+            </div>
+        </div>
 
-        <a href="{% url 'front_end:ctf' %}">
-            <p>CTF</p>
-        </a>
+        <!-- CPCTC -->
+        <div class="col-md-3">
+            <div class="image-container">
+                <a href="{% url 'front_end:cptc' %}">
+                    <img class="logo" src="{% static 'img/cptc.jpg' %}" alt="CPTC" class="img-fluid">
+                    <div class="overlay">
+                        <p>CPTC Team</p>
+                    </div>
+                </a>
+            </div>
+        </div>
     </div>
+</div>
 
-    <script>
-        function show(){
-            var clubsDiv = document.getElementById('clubs');
-            var buttonDiv = document.getElementById('button')
-            clubsDiv.style.display = 'block';
-            buttonDiv.setAttribute("hidden", "hidden")
-        }
-    </script>
 
-</div>
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Same issue here. It's good to keep the scripts in base.html if all pages share them. It might make sense to put a script directly into a template if it's only used in that one template, but it adds clutter.

Reply:
Thank you for letting me know!